foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/datetime/LastFilter.java

```diff
@@ -56,9 +54,36 @@ public TimeWindow getWindow() {
     }
 
     @Override
-    @JsonIgnore
-    public boolean isFilterTemporal() {
-        return true;
+    public int hashCode() {
+        int result = getOperator().hashCode();
```

Comment:
Needed to exclude timestamp field from hashcode for caching to work. Caching doesn't work if the same query gets executed in gap of less than 30s